notebooks/misc/dp_mixgauss_truncated.ipynb

```diff
@@ -0,0 +1,740 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    import optax
</pre><p>Please follow the style guidelijnes listed here:</p><p>https://github.com/probml/pyprobml/tree/master/notebooks</p><p>In particular use</p><pre class="ql-syntax" spellcheck="false">try:
    import optax
except ModuleNotFoundError:
    %pip install -qq optax
    import optax
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/924/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>